src/main/java/com/payline/payment/samsung/pay/service/AbstractPaymentHttpService.java

```diff
@@ -188,8 +189,15 @@ public PaymentResponse processDirectResponse(T paymentRequest, StringResponse re
                         .withExpirationDate(YearMonth.of(decryptedCard.getExpiryYear(), decryptedCard.getExpiryMonth()))
                         .build();
 
+                PaymentData3DS paymentData3DS = PaymentData3DS.Data3DSBuilder.aData3DS()
+                        .withEci(decryptedCard.getEciIndicator())
+                        .withCavv(decryptedCard.getCryptogram())
+//                        .withCavvAlgorithm()
```

Comment:
Sonar S125 - Sections of code should not be "commented out"

Si vous avez pas le cavvAlgorithm, il faut supprimer la ligne.